web/src/CityContentSwitcher.tsx

```diff
@@ -106,7 +111,7 @@ const CityContentSwitcher = ({ cities, match, location }: PropsType): ReactEleme
     )
   }
 
-  const cityRouteProps: CityRouteProps = { cities, languages, cityModel, languageModel }
+  const cityRouteProps = { cities, languages, cityModel, languageModel, pathname, cityCode, languageCode }
```

Comment:
I know you are using implicit types a lot, but I actually think it is not the best way.

Reply:
Why do you think that? :thinking:

Reply:
For example, if adding the type explicitly here, when changing the code later it is clear at the first view, what properties cityRouteProps would need, also if now removing a needed property the error would not show up at this line, but whenever we are using cityRouteProps. I think if the type is not really trivial the pros of explicit typing outweight the cons. At the bottom of this article there is a nice pro/con list:
https://www.schibsted.pl/blog/typescript-type-inference/
But in the end this is something we should agree on a consitent way to do, and nothing specific for this PR.